spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/IntegrationNode.java

```diff
@@ -69,7 +72,7 @@ public Stats getStats() {
 	}
 
 	public Map<String, Object> getProperties() {
-		return this.properties;
+		return this.properties == null ? null : this.properties;
```

Comment:
??? 
```
private final Map<String, Object> properties = new HashMap<String, Object>();
```
So, it can't be `null`. My idea was like:
```
return this.properties.isEmpty() ? null : this.properties;
```